code-env/python/spec/requirements.txt

```diff
@@ -1 +1,2 @@
-kdp-python-connector==0.30.0
\ No newline at end of file
+kdp-python-connector==0.37.6
```

Comment:
This PR is a draft, until the new python connector gets released - https://github.com/Koverse/kdp-python-connector/pull/40